### PR TITLE
Tweaks the 20TC Bundles to be more worthwhile

### PR DESCRIPTION
**Pull request by SconesC**
Nobody ever buys these because they're sort of useless/underwhelming. I personally like being like "Time to invest my 20 TC for 20TC+ of thematic equipment for fun!"

This PR:
- Removes the pathetic ID+voicechanger set, replaces it with the superior Infiltrator bundle
- Adds the False Alarm bundle for making people think it's Nuke (Now with jetpacks!)
- Adds the Takbir bundle, for blowing stuff up and going out with a bang
- Adds several pieces of equipment to bundles, making them a solid investment
- Renames the bundles to make more sense and be less like dumb memes
- Fixes some conventions inconsistency with boxed implants

And yes, I'm aware 'bloodyspai' isn't really removed, just set to prob 0.

### CODE REVIEW

**Comment by Boredone** [comment]
Kinda minor, but I'd suggest tossing the false alarm bundle the Military Belt.
That is very much standard issue for ops, and helps it seem more real.
On the minus side, the Military Belt is kinda robust.

**Comment by SconesC** [comment]
@Boredone I considered that. It completes the disguise but it's also pretty robust.

I guess I'll stick it in there, I love the military belt.

**Comment by Miauw62** [comment]
add a bundle thats just 100 bars of soap

**Comment by GoonOnMoon** [comment]
This is horrible. Please DNM.

The purpose of the 20TC crate is to provide a completely random, unbiased bundle.

**Comment by Miauw62** [comment]
please dont make comments before you carefully read the pr

**Comment by Xhuis** [comment]
Sounds nice. A lot of these bundles aren't worth the telecrystal cost, and this remedies that somewhat. I can see an issue with people using the False Alarm bundle to accomplish the jetpack theft objective, though.

**Comment by Dorsisdwarf** [comment]
I agree. a bundle which is a box full of boxes, and each box contains a wallet, and each wallet contains a bar of soap

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -73,7 +84,7 @@
 			new /obj/item/weapon/aiModule/toyAI(src)
 			return
 
-		if("lordsingulo")
```
Comment (Dorsisdwarf) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: ALL HAIL LORD SINGULO :rotating_light:

**Comment by KorPhaeron** [comment]
The jetpack theft objective is a joke anyway, an emag will accomplish it in about thirty seconds.

Doesn't make much IC sense either considering the syndicate seem to own plenty of jetpacks for their operatives already anyway.

**Comment by Durkel** [comment]
About time someone looked at the bundles, most of them were garbage.

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: don't you dare remove this name.

*grumps*

**Comment by SconesC** [comment]
Taking further requests

Thanks for pointing out the jetpacks, I might make a seperate PR pruning stupid objectives like jetpacks tbh

**Comment by Czarcastic** [comment]
A bundle with a random assortment of grenades would be cool.

**Comment by Niknakflak** [comment]
Make a bundle with 100 bars of soap

**Comment by tgstation-server** [comment]
Make a bundle with 100 bars of soap

**Comment by Anonus** [comment]
No, better; make a grenade that fires bars of soap, then put it in a clusterbang grenade, and give them seven of those.

**Comment by SconesC** [comment]
Yes it's funny but no people shouldn't have a chance for their 20TC thematic kit to be entirely soap

**Comment by Durkel** [comment]
I'd like to ask that you look at all the bundles value wise and evaluate them. Most of them barely break even of the 20 tc cost.

**Comment by Marflowerose** [comment]
Did you add stuff to Sith Bundle? Currently the items it gives are either purely cosmetic and below 20TC while being completely useless.

**Comment by PKPenguin321** [comment]
why ;-;

**Pull request closed without merging**